src/Microsoft.DotNet.XHarness.CLI/Commands/Android/AndroidTestCommand.cs

```diff
@@ -49,7 +50,7 @@ internal class AndroidTestCommand : TestCommand
 Arguments:
 ";
 
-        public AndroidTestCommand() : base(CommandHelp)
+        public AndroidTestCommand() : base("test", true, CommandHelp)
```

Comment:
The second argument was false before, could you please explain, why did you change it?

Reply:
Ah, good catch!

Reply:
Thanks, I suggested two more changes based on that